### PR TITLE
More JS escapes

### PR DESCRIPTION
**Pull request by davidfischer**
Escape additional JS variables in `readthedocs-data.js`.

### CODE REVIEW

**Comment by davidfischer** [comment]
There's some problem with this code. I am having trouble running the unit tests and it worked fine building actual docs but I'll dive a bit deeper.

**Comment by davidfischer** [comment]
This is good to review as I fixed the other issues. I'll see if I can fix the test runner (master is failing) so we can see green test cases.

File: readthedocs_ext/mixins.py [hunk 1]
```diff
@@ -50,11 +50,17 @@ def copy_static_readthedocs_files(self):
                     context=ctx,
                 )
             else:
+                if sphinx.version_info < (1, 5):
```
Comment (stsewd) [inline readthedocs_ext/mixins.py]: This check is already done in the outer if, never is going to reach this.

Comment (davidfischer) [reply readthedocs_ext/mixins.py]: Wow. How'd I miss that!

File: readthedocs_ext/mixins.py [hunk 2]
```diff
@@ -50,11 +50,14 @@ def copy_static_readthedocs_files(self):
                     context=ctx,
                 )
             else:
+                from sphinx.util.template import SphinxRenderer
```
Comment (ericholscher) [inline readthedocs_ext/mixins.py]: Why do we need to use this renderer? Could use a comment.

Comment (davidfischer) [reply readthedocs_ext/mixins.py]: This is just the new way. There's a comment about it here: https://github.com/readthedocs/readthedocs-sphinx-ext/blob/7b6c62b298d7b05a83550e089fc5672f5def5e44/readthedocs_ext/readthedocs.py#L302-L305

**Comment by agjohnson** [comment]
This one has sat around for a while and is in danger of rotting. Let's plan to get a release cut here for one of the next site releases. If we're hesitant to merge this due to potential fallout, let's figure out a QA plan and get this released.

**Pull request merged**